pom.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>CanisLD</groupId>
+  <artifactId>canis-refi</artifactId>
+  <version>1.0-SNAPSHOT</version>
```

Comment:
should main have a different version? that keeps track of the current latest version?

Reply:
probably major versions, then we cut the version for a release.